### DISCUSSION TITLE
Cannot find firmware update as indicated by the helper tool

### DISCUSSION THREAD

**Discussion by Kruijd** [Mr-MIBonk/M.I.B._More-Incredible-Bash#300] [Q&A]
Hi all!
First of all my my compliments for MR MIBonk for all support available  for MIB systems and the easy to use helper tool !
It was very easy to find an update for my MIB system wit partnumber 5G0035846A,  I looked up the SW train, **MSTD_EU_VW_P3350**   but the update suggested (MSTD_EU_VW_P5301, MU  0486) is nowhere to be found on the mibsolution.one site.  
Can anybody help me or give me some directions?

Thanks a lot in advance.
Kind regards, Jan.

**Comment by harman-f** [comment]
Look in pre_MIB2

**Reply by Kruijd** [reply]
Thanks for your reply @harman-f , unfortunately I cannot find the file there, the only MSTD file in this folder is this one:
![image](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/assets/22920122/39e650e3-b393-4601-bb94-45215b5eaa12)

**Reply by RayJay05** [reply]
Hi, Sorry late to the party. You Can find it in this path - MQB_Solution>_pre MIB2>MSTD>VW. Its there now (2026)

**Reply by Kanskov** [reply]
hello i know this post is very old. but i got the MSTD_EU_VW_P5301 update https://drive.google.com/file/d/1I_qxZYPPmQ4ajZNGge9MakjeAqSoC5fd/view?usp=sharing

**Comment by mehdisat1616** [comment]
firmware update: MSTD_EU_VW_P5301

**Reply by Kanskov** [reply]
hello i know this post is very old. but i got the MSTD_EU_VW_P5301 update https://drive.google.com/file/d/1I_qxZYPPmQ4ajZNGge9MakjeAqSoC5fd/view?usp=sharing
